dive_core/common/emulate_pm4.h

```diff
@@ -214,7 +214,7 @@ class EmulatePM4
         kTotalIbLevels
     };
 
-    static const uint32_t kMaxPendingIbs = 100;
+    static const uint32_t kMaxPendingIbs = 150;
```

Comment:
it is normal to have more ibs since on a7xx, each tile has it's own ib

Reply:
randomly chosen :p in this specific case, the max pending is around 130, put 150 just for safety. it is larger than before due to one of my changes to support the CP_START_BIN for a7xx, each bin now has its ib, in this capture there are 66 bins. (Some other PM4s will add pending ibs)